CODE_OF_CONDUCT.md

```
@@ -0,0 +1,128 @@
+
+This Code of Conduct applies within all community spaces, and also applies when
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+shafayet.sadi@gmail.com.
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)